### PR TITLE
added check if the services has started

### PR DESCRIPTION
**Pull request by menkar3**
just try to use CTRL+C when user pass is required in case of "sudo" (or you can fail all password input chances)

### CODE REVIEW

File: autosploit/main.py [hunk 1]
```diff
@@ -29,17 +29,17 @@ def main():
...
             if choice.lower().startswith("y"):
                 if "postgre" in service:
                     cmdline("sudo bash {}".format(START_POSTGRESQL_PATH))
                 else:
                     cmdline("sudo bash {}".format(START_APACHE_PATH))
-                info("service started successfully")
             else:
                 error("service {} is required to be started for autosploit to run, exiting".format(service.title()))
                 sys.exit(1)
+        info("services started successfully")
```
Comment (menkar3) [inline autosploit/main.py]: need "service started successfully", sorry

**Comment by Ekultek** [comment]
What about you @NullArray

File: autosploit/main.py (hunk 1 above)
Comment (Ekultek) [reply autosploit/main.py]: Nah services will for fine

**Comment by Ekultek** [comment]
I'm going to go ahead and merge this since it's a pretty minor update

**Pull request merged**